Essentials/src/com/earth2me/essentials/signs/SignDisposal.java

```diff
@@ -12,8 +14,21 @@ public SignDisposal() {
     }
 
     @Override
-    protected boolean onSignInteract(final ISign sign, final User player, final String username, final IEssentials ess) {
-        player.getBase().openInventory(ess.getServer().createInventory(player.getBase(), 36, tl("disposal")));
+    protected boolean onSignInteract(final ISign sign, final User player, final String username, final IEssentials ess) throws SignException, ChargeException {
+        if (!sign.getLine(3).isEmpty()) {
+            Trade charge = getTrade(sign, 3, ess);
+            charge.isAffordableFor(player);
+            charge.charge(player);
+        }
+
+        StringBuilder titleBuilder = new StringBuilder();
+        titleBuilder.append(sign.getLine(1));
+        titleBuilder.append(sign.getLine(2));
```

Comment:
If the user is not careful, this will smash lines 1 and 2 together - we should add a space between lines 1 and 2 if both have content (and line 1 doesn't end with a space and line 2 doesn't start with a space).

Reply:
I didn't add space on concatenating lines on purpose, in case user *means* to have title twice as long as a single line and not two separate.

Reply:
But then my point is that is inconsistent. While there is a space on the sign, there will not be in the Disposal title. If the users would want a space, they would obviously have to add it to either of the lines, causing the text to then be off-center.